language/move-binary-format/src/errors.rs

```diff
@@ -169,8 +169,30 @@ impl VMError {
             offsets,
         )
     }
+
+    pub fn to_partial(self) -> PartialVMError {
```

Comment:
This function was sort of intentionally missing. What case did you need this for?

Reply:
There was a case where I needed to call a public API from internal, but I don't recall which, so I removed this for now. May need to bring it back when I need in the async PR. In general, I think the types here (and elsewhere) are overly strict, but that is just my taste.